crates/astria-proto/proto/astria/sequencer/v1alpha2/validation.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package astria.sequencer.v1alpha2;
+
+// A merkle proof of inclusion of a leaf at `index` in a tree
+// of size `num_leaves`.
+message InclusionProof {
+    // leaf index of value to be proven
+    uint64 index = 1;
+    // total number of leaves in the tree
+    uint64 num_leaves = 2;
+    // the merkle proof itself; must be 32 bytes.
```

Comment:
merkle proof is probably longer than 32 bytes, should just be `bytes`

Reply:
True, I misread the ctmerkle inclusion proof.

Reply:
i don't see it updated?

Reply:
I have removed `32 bytes` and added that the merkle proof is derived from an RFC 6962 compliant merkle tree.